core-services/prow/02_config/openshift/machine-config-operator/_pluginconfig.yaml

```diff
@@ -12,6 +12,10 @@ label:
     - allowed_users:
       - rioliu-rh
       - sergiordlr
+      - anuragthehatter
+      - rbbratta
+      - zhaozhanqi
+      - huiran0826
```

Comment:
@zhaozhanqi Is there a reason for this list does not have all the team members?

Reply:
Agree with @zhaozhanqi , I was wondering if can we keep this list minimal(maybe 1 or 2 members)? MCO has code owned by few more teams and this list will keep growing and would be difficult to keep track.

Reply:
suggest to add 2 members per team (1 backup)

Reply:
I updated with two sdn members

Reply:
Thanks @zhaozhanqi !